### PR TITLE
Set main as release branch

### PR DESCRIPTION
**Pull request by sguilliaci**

### CODE REVIEW

File: pkg/services/testdata/references.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ workflows:
                     filters:
                         branches:
                             only:
-                                - master
+                                - main
```
Comment (rlegan) [inline pkg/services/testdata/references.yml:17]: ?

File: pkg/services/testdata/references.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ workflows:
...
```
Comment (rlegan) [inline pkg/services/testdata/references.yml:26]: ?

File: pkg/services/testdata/references.yml (hunk 1 above)
Comment (AbrahamTewa) [reply pkg/services/testdata/references.yml:17]: LGTM. Why the "?" @rlegan ?

Comment (rlegan) [reply pkg/services/testdata/references.yml:17]: why this file ? its a test file

Comment (sguilliaci) [reply pkg/services/testdata/references.yml:17]: @rlegan mass replace :trollface:

**Pull request closed without merging**